mlflow/projects/utils.py

```diff
@@ -5,10 +5,8 @@
 import urllib.parse
 import pathlib
 
-from distutils import dir_util
+from distutils import dir_util  # pylint: disable=W0402
```

Comment:
Can we avoid using a deprecated module?

Reply:
Done. replaced by shutil

Reply:
I somehow thought you added this line, but you just added a comment. Why did we need this comment?